osm2gtfs.py

```diff
@@ -82,28 +92,28 @@ def main():
 
     schedule = transitfeed.Schedule()
     agency = schedule.AddAgency(
```

Comment:
I like this a lot! For now (in this pull request), I just introduced the config file which also has the same data structure. So, we can definitively use this call in the code to generate the agency gtfs information nicely.